tests/view/crudeapi/test_crude_api.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import werkzeug
+from timeless.views import CrudAPIView
+from timeless.restaurants.models import Location
+from timeless.companies.models import Company
+"""
+    Tests for CrudeAPIView.
```

Comment:
@paulodamaso I think it should be ``CrudAPIView``, right? Without the ``e``. And also in the filename: ``test_crud_api.pi``, without ``e``?

Reply:
@amihaiemil Done!